installer/install.sh

```
@@ -0,0 +1,635 @@
+}
+
+setup_cloudflare_tunnel_full_auto() {
+    echo "Starting full-automatic Cloudflare Tunnel setup..."
+    install_packages jq
+
+    ACCOUNTS_DATA=$(curl -s -X GET "https://api.cloudflare.com/client/v4/accounts" \
+         -H "X-Auth-Email: $CF_EMAIL" \
+         -H "X-Auth-Key: $CF_API_KEY" \
+         -H "Content-Type: application/json")
+
+    ACCOUNT_COUNT=$(echo $ACCOUNTS_DATA | jq -r '.result | length')
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)